examples/delegator/lib.rs

```diff
@@ -14,7 +14,6 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
-pub use self::delegator::Delegator;
```

Comment:
I think it is fine that this PR deletes this line of code but in general we might need some strategy to expose public items from a smart contract. E.g. for `cargo doc`, otherwise it will display an empty smart contract API if there are no public items.